tests/acceptance/features/apiSpacesShares/shareSubItemOfSpace.feature

```diff
@@ -146,3 +146,30 @@ Feature: Share a file or folder that is inside a space
     Then the HTTP status code should be "200"
     And the information about the last share for user "Brian" should include
       | expiration |  |
+
+
+  Scenario: check the end of expiration date in user share
+    Given user "Alice" has created a share inside of space "share sub-item" with settings:
+      | path       | folder                   |
+      | shareWith  | Brian                    |
+      | role       | viewer                   |
+      | expireDate | 2042-01-01T23:59:59+0100 |
+    And user "Brian" has accepted share "/folder" offered by user "Alice"
+    When user "Alice" expires the last share
```

Comment:
Might be good if we can be specific here.
```suggestion
    When user "Alice" expires the share "/folder" for user "Brian"
```

Reply:
yes but I don't think we would use "<resource-name>" and "<user>" in script for updating share because we get shareID from  `$shareId = $this->featureContext->getLastPublicLinkShareId();`  which get's stored when the share is created. 

We could send a request to get the shareID by providing the above ^ info but that seems pointless because we already get the id from `$this->featureContext->getLastPublicLinkShareId();`.

We also follow a similar pattern for 
`user updates the last share with settings:`

Reply:
Should be fine with current step though.
I was only cautious regarding some cases where we might need to create multiple shares and then update some of them.
But looks good to me 👍🏻